pkgs/development/libraries/libcbor/default.nix

```
@@ -2,13 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libcbor";
-  version = "2019-02-23";
+  version = "2019-07-25";
```

https://nixos.org/nixpkgs/manual/#sec-package-naming 
says to append unstable to name, but for the case of pname, i think it makes more sense to prefix version. 
```suggestion
  version = "unstable-2019-07-25";
```

---

Hmm, okay.  Thanks!